src/wave.js

```diff
@@ -73,14 +73,32 @@ class WaveBase {
         window.addEventListener('resize', () => {this.handleResize()}, false);
     }
 
-    initCamera() {
+    initOrthographicCamera() {
         const initialPosition = [-50, 0, 10];
-        this.camera = new THREE.PerspectiveCamera(20, this.ASPECT, 1, 20000);
-        this.camera.name = "PerspectiveCamera";
-        this.camera.position.copy(new TV3(...initialPosition));
-        this.camera.up = new TV3(0, 0, 1);
-        this.camera.lookAt(new TV3(0, 0, 0));
-        this.scene.add(this.camera);
+        this.orthographicCamera = new THREE.OrthographicCamera(-10 * this.ASPECT, 10 * this.ASPECT, 10, -10, 1, 1000);
+        this.orthographicCamera.name = "OrthographicCamera";
+        this.orthographicCamera.position.copy(new TV3(...initialPosition));
+        this.orthographicCamera.up = new TV3(0, 0, 1);
+        this.orthographicCamera.lookAt(new TV3(0, 0, 0));
+        this.scene.add(this.orthographicCamera);
+    }
+
+    initPerspectiveCamera() {
+        const initialPosition = [-50, 0, 10];
```

Comment:
Also, we might be using this position in multiple places - move to settings?

Reply:
Done.